### DISCUSSION TITLE
Intervention\Image v2 is no longer actively supported

### DISCUSSION THREAD

**Discussion by Kovah** [awcodes/filament-curator#540] [General]
I just wanted to use the Intervention Image package and noticed that I still had v2 installed, but the feature I need is only available in v3. Unfortunately, filament-curator requires v2 to be installed.

Would it be possible to get filament-curator to use Image v3 soon?

The v2 package [is no longer supported](https://image.intervention.io/v2) and can probably opens up the application to security vulnerabilities.

**Comment by awcodes** [comment]
Unfortunately, the Glide package is still locked to v2 so I can't do much about it.

**Comment by james2doyle** [comment]
V3 is on the horizon: https://github.com/thephpleague/glide/tree/3.x/docs/3.0

**Reply by marc31** [reply]
Glide 3 use intervention v3 now https://github.com/thephpleague/glide/blob/3.0.1/composer.json

**Reply by Kovah** [reply]
Your posts are about Glide, but Curator uses the glide-symfony package specifically. Still waiting for a release there.

**Reply by awcodes** [reply]
I have glide v3 with intervention v3 working in the next major version of curator. But it's not just about glide. Curator uses the intervention package outside of just glide so it not a straight forward upgrade.
